sopel/modules/wikipedia.py

```diff
@@ -272,18 +272,74 @@ def mw_section(server, query, section):
     return text
 
 
+def say_image_description(bot, trigger, server, image):
+    desc = mw_image_description(server, image)
+
+    if desc:
+        bot.say(desc, truncation=" […]")
+
+
+def mw_image_description(server, image):
+    """
+    Retrieves the description for the given image
+    """
```

Comment:
Nit, matching style elsewhere in the file:

```suggestion
    """Retrieves the description for the given image."""
```